listening-platform/src/test/kotlin/com/nikitahohulia/listeningplatform/NatsMovieControllerTests.kt

```diff
@@ -93,90 +99,84 @@ class NatsMovieControllerTests {
 
     @Test
     fun getUserByIdTestOk() {
-        userRepository.save(userToAdd)
+        val user = userRepository.save(getUserToAdd()) ?: throw MongoException("Failed to save user")
 
-        val message = GetUserByIdRequestCommon.newBuilder().setUserId(userId).build()
+        val message = GetUserByIdRequest.newBuilder().setUserId(user.id?.toHexString()).build()
 
         val actual = doRequest(
             GET_BY_ID,
             message,
-            GetUserByIdResponseCommon.parser()
+            GetUserByIdResponse.parser()
         )
 
-        val expected = GetUserByIdResponseCommon.newBuilder().apply {
+        val expected = GetUserByIdResponse.newBuilder().apply {
             successBuilder
-                .setUser(userToAdd.toProto())
+                .setUser(user.toProto())
         }.build()
 
         assertThat(actual).isEqualTo(expected)
     }
 
     @Test
     fun updateUserTestOk() {
-        userRepository.save(userToAdd)
+        val user = userRepository.save(getUserToAdd()) ?: throw MongoException("Failed to save user")
+        val updatedProto = getUserToAdd().copy(id = user.id, publisherId = user.publisherId).toProto()
 
-        val proto = userToAdd.copy(
-            username = "Updated",
-            email = "new.update.com",
-            publisherId = ObjectId("651c6a8763d50fb3f7f1ec7c")
-        ).toProto()
+        val message = UpdateUserRequest.newBuilder().setUser(updatedProto).build()
 
-        val message = UpdateUserRequestCommon.newBuilder().setUser(proto).build()
-
-        val actual = doRequest(UPDATE, message, UpdateUserResponseCommon.parser())
-        val expected = UpdateUserResponseCommon.newBuilder().apply {
-            successBuilder.setUser(proto)
+        val actual = doRequest(UPDATE, message, UpdateUserResponse.parser())
+        val expected = UpdateUserResponse.newBuilder().apply {
+            successBuilder.setUser(updatedProto)
         }.build()
 
         assertThat(actual).isEqualTo(expected)
     }
 
     @Test
     fun getUserByUsernameTestOk() {
-        userRepository.save(userToAdd)
+        val user = userRepository.save(getUserToAdd()) ?: throw MongoException("Failed to save user")
```

Comment:
```suggestion
        val user = userRepository.save(getUserToAdd())!!
```
in tests, it's applicable to use !! as you know that user will be returned